src/news/2020-1-galaxy-release/index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: 'Galaxy Release 20.01'
+tease: Workflow Reporting, Workflow Executions, Screenreader-friendly navigation
+date: '2019-02-27'
```

Comment:
2020 :)

Reply:
haha, good catch.

Reply:
that would have been really on time though ;)